gravitee-plugin-core/src/main/java/io/gravitee/plugin/core/internal/PluginImpl.java

```diff
@@ -82,6 +85,18 @@ public void setDependencies(URL[] dependencies) {
         this.dependencies = dependencies;
     }
 
+    public void setTimestamp(long timestamp) {
```

Comment:
You can replace setter/getter by lombok annotation when you can.

Reply:
No lombok configured anywhere in gravite-plugin-*, I kept as is

Reply:
We have informally decided to migrate when we touch some code.

Reply:
Oh OK. Then I need to setup/enable some maven stuff, because I reverted since it was not building. I'll look into it